mappings/net/minecraft/client/render/Camera.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS ctf net/minecraft/client/render/Camera
+	FIELD a ready Z
+	FIELD b area Lbfi;
+	FIELD c focussedEntity Lahw;
```

Comment:
`focusedEntity`

Reply:
I even googled this to check the spelling but alas